ddp-workspace/projects/ddp-sdk/src/lib/components/prism/prism.component.html

```diff
@@ -25,6 +25,13 @@
         </p>
     </ng-container>
 </div>
+<mat-paginator #paginator
+               [hidden]="isSearchLoading || dataSource.data.length <= 10"
+               [pageSizeOptions]="[10, 25, 100]"
```

Comment:
#optional
May be use this value `[10, 25, 100]` as a constant ?

Reply:
good idea! done